Csstudio 2963 grouping replies bug

maxfrederiksen-ess opened the pull request:
## Summary of Changes

- Reverted to the old logic of grouping "replies"
- Changed the terminology from replies to group in the expand button
- Added new reply icon and design temporarily (will hopefully get feedback on that from balazs) but the logic, ordering behavior is how I want it right now at least
- Reduced size of the replies

georgweiss reviewed (approved):
Are the changes just for the beta UI?

maxfrederiksen-ess commented:
> Are the changes just for the beta UI?

Yes. This is one part in a series of tasks to get the grouping/sorting behavior like what we have discussed in our meetings. So this task makes sure that the newest reply to a group is first, and the original is on the bottom. I also modified the design a little bit to fit more this behavior, but will run it by you and balazs after I've fixed the following tasks as well.
